docs/jwst/outlier_detection/index.rst

```diff
@@ -2,6 +2,52 @@
 Outlier Detection
 =================
 
+Processing multiple datasets together allows for the identification of bad pixels
+or cosmic-rays that remain in each of the input images, many times at levels which
+were ambiguous for detection during ramp fitting.  The outlier detection step 
+implements the following algorithm to identify and flag any remaining cosmic-rays or
+other artifacts left over from previous calibrations:
+
+  - build a stack of input data
+  
+    - all inputs will need to have the same WCS in order
```

Comment:
Not sure what the phraseology "the same WCS in order" means? Or did a typo happen?